gadget/update.go

```diff
@@ -61,11 +61,16 @@ type ContentChange struct {
 }
 
 type ContentOperation int
+type ContentChangeDisposition int
```

Comment:
we'll need a native speaker input here:  should this be:

* ...ChangeDisposition
* ...ChangeResolution
* ...ChangeDecision
* ...ChangeAction 

or yet something else?

@anonymouse64 or @degville any input on this?   

the values of this can be like: apply, abort, preserve as seen below

Reply:
Action sounds the most natural to me I think

Reply:
> Action sounds the most natural to me I think

Yes, I agree.

Reply:
Thanks for the input. ChangeAction it is then.